core/controllers/admin.py

```diff
@@ -111,7 +111,7 @@ def get(self):
             'role_graph_data': role_services.get_role_graph_data()
         })
 
-        self.render_template('pages/admin/admin.html')
+        self.render_template('pages/admin/admin.mainpage.html')
```

Comment:
Make this as admin-page. Also, make the folder name as admin-page as is the convention.